SmartVision/WindowsForms/FaceDetection.cs

```diff
@@ -1,15 +1,18 @@
 ﻿using System;
+using System.Drawing;
 using System.Windows.Forms;
 using Emgu.CV;
 using Emgu.CV.Structure;
 
 namespace WindowsForms
 {
-    public partial class Form1 : Form
+    public partial class FaceDetection : Form
```

Comment:
Since it is a **Form**, the name of the class should somehow reflect it in my opinion, not sure what the convention is, but for example **FormFaceDetection** or something of the sort.